src/applications/travel-pay/containers/SubmitFlowWrapper.jsx

```diff
@@ -169,13 +168,6 @@ const SubmitFlowWrapper = () => {
           {!isUnsupportedClaimType &&
             !submissionError &&
             pageList[pageIndex].component}
-          <div className="vads-u-margin-top--4">
-            <va-need-help>
-              <div slot="content">
-                <HelpTextManage />
-              </div>
-            </va-need-help>
-          </div>
```

Comment:
@liztownd Is this the one that needs to stay and the intro page one that should be removed?
After you mentioned this in the demo I thought we might want this to stay.